src/rules/no-restricted-patterns.ts

```diff
@@ -1,36 +1,43 @@
 import * as gherkinUtils from './utils/gherkin';
-import {GherkinData, RuleSubConfig, RuleError, GherkinNode} from '../types';
-import {Background, Scenario, Step} from '@cucumber/messages';
+import { GherkinData, RuleSubConfig, RuleError, GherkinNode } from '../types';
+import { Background, Scenario, Step } from '@cucumber/messages';
 
 type IConfiguration<T> = {
   Global: T[]
   Scenario: T[]
   ScenarioOutline: T[]
   Background: T[]
   Feature: T[]
+  Given: T[]
+  When: T[]
+  Then: T[]
 };
 
+const Keywords = ['Given', 'When', 'Then'];
```

Comment:
Please keep coding style consistency, variables must be in camelCase, starting with lowercase
```suggestion
const keywords = ['Given', 'When', 'Then'];
```